Config/Project64.rdb

```diff
@@ -8133,4 +8113,4 @@ Good Name=MGC 2011 Demo (PD)
 Internal Name=shut up and code
 CPU Type=Interpreter
 RDRAM Size=8
-Status=Unsupported
+Status=Unsupported
```

Comment:
noooooooooooooooo!

That poor end-of-file line break :sob: , slaughtered like the helpless sacrificial semantic entity that he is.